### PR TITLE
[BBC-9097] Snackbar.MDX

### PR DESCRIPTION
**Pull request by pith**
## Description

Migrate the snackbar component to MDX.

## What has been done

image

## Things to consider

I also changed few tests to RTL. I don't know if we can test the close button with the new MDX format.

## How it was tested

Storybook, unit test.

### CODE REVIEW

File: src/snackbar/Snackbar.story.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+### Animation
+
+To come soon (appears from bottom.)
+
+## Usage
+
+```jsx
+import { Snackbar } from '@blablacar/ui-library/build/Snackbar'
+;<Snackbar />
```
Comment (mcellier) [inline src/snackbar/Snackbar.story.mdx:56]: wrong import path.

File: src/snackbar/Snackbar.tsx [hunk 2]
```diff
@@ -9,13 +9,14 @@ import { Button, ButtonStatus } from '../button'
...
 import { AnimationType, Transitions as CustomTransition } from '../transitions'
+import { StyledSnackbar } from './Snackbar.style'
 
 export type SnackbarProps = Readonly<{
   close: () => void
   isOpen: boolean
   className?: string
   extraClassName?: string
-  children: JSX.Element
+  children: JSX.Element | string
```
Comment (mcellier) [inline src/snackbar/Snackbar.tsx:19]: `React.ReactNode` instead of `JSX.Element | string`

**Review by mcellier — APPROVED** [review]

File: src/snackbar/Snackbar.story.mdx (hunk 1 above)
Comment (mcellier) [reply src/snackbar/Snackbar.story.mdx:56]: - [ ] Fix path

File: src/snackbar/Snackbar.tsx (hunk 2 above)
Comment (mcellier) [reply src/snackbar/Snackbar.tsx:19]: - [ ] Fix type

**Comment by wakooka** [comment]
You have an extra commit that should not be thre.

**Pull request merged**